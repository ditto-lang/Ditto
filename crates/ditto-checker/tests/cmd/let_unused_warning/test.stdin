module Test exports (..)

why = let fives : Array(Int) = [5] in unit
