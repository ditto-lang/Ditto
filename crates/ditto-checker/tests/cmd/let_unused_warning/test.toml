bin.name = "ditto-checker-testbin-check-module"
args = ["let_unused_warning.ditto"]
fs.sandbox = true
