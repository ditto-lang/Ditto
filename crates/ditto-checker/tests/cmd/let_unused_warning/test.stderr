
  ⚠ unused let binder
   ╭─[let_unused_warning.ditto:2:1]
 2 │ 
 3 │ why = let fives : Array(Int) = [5] in unit
   ·           ──┬──
   ·             ╰── this isn't used
   ╰────

